Tests/scripts/lock_cloud_machines.py

```diff
@@ -294,8 +295,9 @@ def create_list_of_machines_to_run(storage_bucket, lock_machine_name, gcs_locks_
         list_machines = [lock_machine_name]
     else:
         logging.info('getting all machine names')  # We are looking for a free machine in all the available machines.
-        test_machines_list = storage_bucket.blob(f'{gcs_locks_path}/{test_machines_path}')
-        list_machines = test_machines_list.download_as_string().decode("utf-8").split()
+
+        list_machines = list(json.loads(test_machines).keys())
```

Comment:
I think It is better to use $TEST_MACHINES_LIST_STRING, and then no json loads needed.